spec/models/date_parser_spec.rb

```diff
@@ -27,6 +27,12 @@
     it { is_expected.to eql(Date.new(2020, 1, 9)) }
   end
 
+  context "when given a hash containing a negative year" do
+    let(:input) { { year: "-2020", month: "1", day: "32" } }
+
+    it { is_expected.to eq(OpenStruct.new(year: "-2020", month: "1", day: "32")) }
+  end
```

Comment:
Suggest changing the day to a valid one, as otherwise this would pass anyway as the date is invalid.  Also suggest making the month or day a negative number instead of the year, as negative years are valid. It's unlikely that we'd need to support BC dates, but these can be validated through a separate validator (eg we already have `validate :assessed_on_cannot_be_older_than_1970` on the risk assessment form).

```suggestion
  context "when given a hash containing a negative year" do
    let(:input) { { year: "2020", month: "-2", day: "30" } }

    it { is_expected.to eq(OpenStruct.new(year: "2020", month: "-2", day: "30")) }
  end
```